src/components/NavBar/store.js

```diff
@@ -1,20 +1,80 @@
-import { observable, action } from 'mobx';
+import { observable, action, computed } from 'mobx';
+import { OracleStatus, Token } from 'constants';
+import _ from 'lodash';
+import Topic from '../../stores/models/Topic';
+import Oracle from '../../stores/models/Oracle';
+
+import { queryAllTopics, queryAllOracles, queryAllVotes } from '../../network/graphQuery';
 
 const INIT_VALUES = {
   myActivitesCount: 0, // INIT_VALUESial loaded state
 };
 
 
-export default class NavBarStore {
-  @observable myActivitesCount = INIT_VALUES.myActivitesCount
+export default class {
+  @observable ResultSettingCount = INIT_VALUES.ResultSettingCount
+  @observable FinalizeCount = INIT_VALUES.FinalizeCount
+  @observable WithdrawCount = INIT_VALUES.WithdrawCount
+  @computed get totalCount() {
+    return this.ResultSettingCount + this.FinalizeCount + this.WithdrawCount;
+  }
 
-  @action.bound
-  async init() {
-    // this.myActivitesCount = await this.getMyActivitiesCount();
+  constructor(app) {
+    this.app = app;
   }
 
-  // @action.bound
-  // async getMyActivitiesCount() {
+  @action
+  init() {
+    this.getCount();
+  }
+
+  @action.bound
+  async getCount() {
```

Comment:
Can you actually move this to the `GlobalStore`? This call is in `GlobalHub` and the counts are not only used in the `NavBar`. It is also used in the `Activites` component in the tab counts. 

Once moved to GlobalStore, believe you can remove it entirely from the `GlobalHub`.

Reply:
I know other places will use it, but I haven't decided where to put, I think putting it to GlobalStore is bad, I was thinking about putting it to a new store called ActivitiesStore(was thinking to do it in an another PR)

Reply:
I just try to narrow the scope of the refactor PR.

Reply:
I think moving into GlobalHub is a good idea, updated